scripts/triggered_emails.py

```diff
@@ -0,0 +1,67 @@
+import sys
+import logging
+from datetime import datetime, timedelta
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+from website.app import init_app
+from website import mails, settings
+from framework.auth import User
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+def main(dry_run=True):
+    #one query for 6 weeks and osf4m users, and one for 4 weeks for regular users
+    inactive_users = list(User.find(Q('date_last_login', 'lt', datetime.utcnow() - settings.NO_LOGIN_WAIT_TIME) &
+                                    Q('conference_user', 'eq', False)))
+    inactive_users.extend(list(User.find(Q('date_last_login', 'lt', datetime.utcnow() - settings.NO_LOGIN_WAIT_TIME - timedelta(weeks=2)) &
```

Comment:
Is NO_LOGIN_WAIT_TIME specifically for osf4m users or non osf4m users? Either way I think they should have their own individual constants and not subtract a timedelta based on an individual config setting. What if for example we want to email users who haven't logged in for three weeks instead of four? Is this the only place using a timedelta like this?